src/themes/custom/app/shared/object-collection/shared/badges/badges.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+import { BadgesComponent as BaseComponent } from 'src/app/shared/object-collection/shared/badges/badges.component';
+
+@Component({
+  selector: 'ds-badges',
+  // styleUrls: ['./badges.component.scss'],
```

Comment:
This file is missing a reference to the default SCSS in `styleUrls`.  It looks like a default `badges.component.scss` file does exist (even though it's empty).